notebooks/Sentinel-2/SAFE_Zarr_Metadata_Comparison_S2_L2A.ipynb

```diff
@@ -6,8 +6,8 @@
    "metadata": {},
    "source": [
     "---\n",
-    "title: SAFE vs Zarr Metadata\n",
-    "subtitle: Learn how to extract metadata from SAFE (Standard Archive Format for Europe) in comparison to the EOPF Zarr using GDAL and Xarray for a sample Sentinel-2 L2A dataset\n",
+    "title: Zarr vs SAFE Metadata\n",
+    "subtitle: Learn to extract Metadata from EOPF Zarr vs. SAFE Using GDAL & Xarray (Sentinel-2 L2A)\n",
```

Comment:
Learn how to extract Meta data from EOPF Zarr, in comparison with legacy SAFE format using Xarray and GDAL ??